content/md/imports-exports/tailored-export.md

```diff
@@ -0,0 +1,86 @@
+---
+id: tailored-export
+themes: import-export-data
+title: "Explore new possibilities with a tailored export"
+popular: false
+ee-only: true
+related: exports, monitor-jobs, access-rights-on-imports-exports
+---
+
+# Overview
+
+This feature consists in a new export profile, available only for our Enterprise Edition or Growth Edition users.
+It helps you to export your product information in flat files, in the ways you want.  
+For instance, let's assume your retailers or distributors need the same product information from you in flat file, but they both want different structures:  
+* Retailer A wants an export with columns named: "Identifier", "Product Name", "Description" and "Weight (cm)".
+* Retailer B wants an export with columns named: "Name in french", "Name in english", "Description 1", "ID" and "Weight (mm)".  
+
+It means you need to export your information thanks to the classic product export, and to manually modify your spreadsheet to reorder the columns, change their name, convert the measurement and other low value-added tasks.  
+Now you can easily create different Tailored Export profiles to match the requirements of most of your retailers or distributors!
+
+# Create a tailored product export profile
+
+To create a product export:
+1. Go to `Exports`
+1. Click on `Create export profile`
+1. Type a unique export code and a label, then select the job `Tailored product export` or `Tailored product model export`. You can select either CSV or XLSX for each job.
+![Create a new product export profile](../img/TailoredExport_Create-export-profile.png)
+1. Save your new export profile
+
+# Edit a tailored product export profile
```

Comment:
as this part is not different from other export profile, I find it confusing: it looks like there is something different that the user need to know.

Reply:
It's useful to new users, so we prefer to keep it like this